### ISSUE TITLE
[request] Force player to spec bot

### ISSUE THREAD

**Issue by mloveless1227** [nikooo777/ckSurf#76]
This is not so much an issue as an inquiry.

I was wondering if it's simple enough or possible for you guys to implement a feature+cvar that, if enabled, would make spectators view the main replay bot  by default? So if they just changed teams to spec or were moved to spec, they would automatically spec the bot instead of another player. They would still have the option to type !spec playername and go to a person, this is just a default action feature I was looking for. 

For example, an afk manager sending someone to spec would force them to view another player, and that player would lose their knife (due to plugin bug) and also have the 'spectators' panel showing to them. Just minor inconveniences on the person being spectated.

**Comment by M-archand** [comment]
Yeah, this would be nice, people seem to get anxious/annoyed when someone is spectating them... who might just be an afk haha
